### DISCUSSION TITLE
Baseball fields have road as the terrain and diamonds and is rooved.

### DISCUSSION THREAD

**Discussion by thelonestander** [cataclysmbn/Cataclysm-BN#2150] [General]
Odd, but when I was flying over the baseball field, I noticed it has road as terrain for the diamonds. Its also rooved. Pics for context below.  
![Screenshot_1293](https://user-images.githubusercontent.com/82045140/199240876-ca4ca8e1-7e56-4904-acad-e364d322c331.png)
![Screenshot_1294](https://user-images.githubusercontent.com/82045140/199240888-010fdc98-6705-439f-beca-292d1f712d5c.png)

**Comment by scarf005** [comment]
hello @thelonestander, thanks for the report. it seems to be pure json problem, and was converted to issue on #2447. thank you!
